### PR TITLE
Delete todo

### PR DESCRIPTION
**Pull request by Jeffersonlpu**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -9,13 +9,18 @@ function App() {
     // TODO
   };
 
+  const handleRemoveTodo = (id: string) => {
+    console.log(id)
```
Comment (jhrodriguesit) [inline src/App.tsx]:
```suggestion
```
Great job @Jeffersonlpu, well done!
Could You please remove this console.log?

**Review by jhrodriguesit — CHANGES_REQUESTED** [review]

File: src/components/Todo/Todo.tsx [hunk 2]
```diff
@@ -48,8 +47,8 @@ const Todo = ({ id }: { id: string }) => {
         <div className="content">
           <span>Checkbox Label</span>
 
-          {true && (
-            <button>
+          {isHovered && (
```
Comment (mortoni) [inline src/components/Todo/Todo.tsx:50]: Thanks for fixing this

File: src/App.tsx [hunk 3]
```diff
@@ -9,13 +9,18 @@ function App() {
...
+    setItems(items.filter((item) => item !== id));
```
Comment (mortoni) [inline src/App.tsx:13]: Excellent!!! you understand it, here is the core of your functionality.

**Review by mortoni — APPROVED** [review]
Great work!

File: src/App.tsx (hunk 1 above)
Comment (Jeffersonlpu) [reply src/App.tsx]: Thanks, @jhrodriguesit ! The console.log was already removed!

File: src/App.tsx (hunk 3 above)
Comment (Jeffersonlpu) [reply src/App.tsx:13]: Thanks, @mortoni !

**Review by jhrodriguesit — APPROVED** [review]
Fantastic Work @Jeffersonlpu !! Well done!

**Pull request merged**